### PR TITLE
Update .Rbuildignore

### PR DESCRIPTION
**Pull request by dan-knight**
- [X] I have read the [code review guidelines](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List).

- [X] The name of the branch is meaningful and well formatted following the [standards](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List), using \[AD_username (or 5 letters of AD if AD is too long)-\[brief_description_of_branch].

- [X] I have set up or verified the branch protection rule following the [github standards](https://confluence.mednet.ucla.edu/pages/viewpage.action?spaceKey=BOUTROSLAB&title=GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.

- [X] I have added the changes included in this pull request to `NEWS` under the next release version or unreleased, and updated the date.

- [X] I have updated the version number in `metadata.yaml` and `DESCRIPTION`.

- [X] Both `R CMD build` and `R CMD check` run successfully.

Expands `.Rbuildignore` to include changes to the lab [R package template](https://github.com/uclahs-cds/template-R-package). This change is also necessary for CRAN resubmission.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,9 @@
+BoutrosLab.plotting.general 7.0.3 2022-03-15
+
+BUG
+* Update .Rbuildignore to match lab R standards
```
Comment (stefaneng) [inline NEWS:4]: Not sure if updating `.Rbuildignore` should be under `BUG`?

**Review by stefaneng — COMMENTED** [review]
Also,

File: NEWS (hunk 1 above)
Comment (dan-knight) [reply NEWS:4]: > Not sure if updating `.Rbuildignore` should be under `BUG`?

Maybe "update" instead? I was unsure about this one. I didn't go with update initially, as it's not a new feature or anything.

Comment (stefaneng) [reply NEWS:4]: Update to me sounds better than bug but your call. Since the first word is "update" it seems like it makes sense :)

Comment (dan-knight) [reply NEWS:4]: Hahah, Good point! Update it is.

**Review by stefaneng — APPROVED** [review]

**Pull request merged**